.rubocop-https---raw-githubusercontent-com-trailblazer-meta-master-rubocop-yml

```diff
@@ -99,3 +100,37 @@ Metrics/AbcSize:
   Max: 25
 Style/LambdaCall:
   Enabled: false
+Style/Semicolon:
+  Enabled: false
+Naming/UncommunicativeMethodParamName:
+  Enabled: false
+Style/ClassAndModuleChildren:
+  Enabled: false
+Layout/LeadingCommentSpace:
+  Exclude:
+    - 'test/docs/**/*'
+Layout/AlignHash:
+  EnforcedHashRocketStyle: table
+Style/FrozenStringLiteralComment:
+  Enabled: false
+Layout/AlignHash:
+  EnforcedColonStyle: table
+SingleLineMethods:
+  Enabled: false
+Style/Lambda:
+  EnforcedStyle: literal
+Style/AsciiComments:
+  Enabled: false
+Style/CollectionMethods:
+  Enabled: false
+Style/RedundantReturn:
+  Enabled: false
+Style/PercentLiteralDelimiters:
+  PreferredDelimiters:
+    default: {}
+    "%q": '()'
+    "%r": '{}'
+    "%w": '[]'
+    "%": '{}'
+Style/HashSyntax:
+  Enabled: false
```

Comment:
I think we have removed rubocop from the repos and just make sure to use it locally